client/my-sites/plans/main.jsx

```diff
@@ -78,9 +73,6 @@ const Plans = React.createClass( {
 							cart={ this.props.cart }
 							selectedSite={ selectedSite } />
 
-						<QueryPlans />
-						<QuerySitePlans siteId={ selectedSiteId } />
```

Comment:
I'm getting infinite loaders on the /plan page. Let's move these 2 query components to PlansFeaturesMain